### ISSUE TITLE
VR Player - support roll in mono videos

### ISSUE THREAD

**Issue by mungewell** [gusmanb/PSVRFramework#43]
Head roll is not represented in the video image.

I understand that there are 'issues' with stereoscopic (3D) clips, but mono should work fine.
Simon

**Comment by gusmanb** [comment]
Yes, I know, I think I wrote about it on the sensor fusion post issue that it's disabled for all videos and I need to activate it on the mono ones.

Along this week I will change the video library to GStreamer and the new version hopefully will have a lot more options, better interface and the roll for mono videos.
